syntax = "proto3";
package process_definition_version;




import "google/protobuf/empty.proto";

/*
DeleteProcessDefinitionVersion请求
*/
message DeleteProcessDefinitionVersionRequest {
    /*
    流程定义实例ID
    */
    string definitionId = 1 ;
    /*
    数据定义版本实例ID(lastest表示最新版本,多个版本用';'分隔)
    */
    string versionIds = 2 ;
}

/*
DeleteProcessDefinitionVersionApi返回
*/
message DeleteProcessDefinitionVersionResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    google.protobuf.Empty data = 4 ;
}

